src/main/java/org/rhizome/server/domain/article/domain/Article.java

```diff
@@ -0,0 +1,47 @@
+package org.rhizome.server.domain.article.domain;
+
+import java.time.LocalDateTime;
+
+import org.rhizome.server.support.BaseTimeEntity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Article extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(columnDefinition = "TEXT")
+    private String content;
+
+    @Column(name = "created_at")
+    private LocalDateTime createdAt;
+
+    @Column(name = "updated_at")
+    private LocalDateTime updatedAt;
+
+    @Builder
+    private Article(String title, String content) {
+        this.title = title;
+        this.content = content;
+    }
+
+    public Article createArticle(String title, String content) {
+        return Article.builder().title(title).content(content).build();
+    }
```

Comment:
정적 팩토리 메서드로 쓸 의도였다면 static이 빠져있습니다
```suggestion
    public static Article createArticle(String title, String content) {
        return Article.builder().title(title).content(content).build();
    }
```

Reply:
반영 완